core/spec/lib/tasks/migrations/delete_prices_with_nil_amount_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+path = Spree::Core::Engine.root.join('lib/tasks/migrations/delete_prices_with_nul_amount.rake')
```

Comment:
if you change the name of the task, the change should be reflected in this file.